app/views/admin.editCategories.html

```diff
@@ -8,15 +8,22 @@
         </tr>
     </thead>
     <tbody>
-        <tr ng-repeat="category in allCategories">
+        <tr ng-repeat="category in categories">
             <td>
                 <img ng-if="!category.image" src="images/noimage.png" height="50px">
                 <img ng-if="category.image" ng-src="{{ category.image }}" height="50px">
             </td>
 
             <td>{{category.name}}</td>
             <td>{{category.description}}</td>
-            <td><button type="button" class="btn btn-success"><a href="#!/admin/editCategory/{{ category.id }}">Edit</a></button> <button type="button" class="btn btn-danger"><a href="#!/deleteCat/{{ category.id }}">Delete</a></button> </td>
```

Comment:
Did I really write this as a single line :( 
Thanks for making it way more legible!

Reply:
Haha we've all done it! :D